mathics/eval/assignments/assignment.py

```diff
@@ -1292,22 +1292,11 @@ def process_rhs_conditions(
     lhs: BaseElement, rhs: BaseElement, condition: Expression, evaluation: Evaluation
 ) -> Tuple[BaseElement, Optional[BaseElement]]:
     """
-    lhs = Condition[rhs, test] -> Condition[lhs, test]  = rhs
+    Add back the lhs conditions.
```

Comment:
The name "process_rhs_conditions" seems to me a misnomer. This function right now doesn't do _anything_ with the parameter, "rhs", except pass it back. 

So, as far as I can tell, the "rhs" parameter might be removed altogether. 

What this function does right now seems to be to _wrap_ `lhs` in a "Condition" expression, if parameter `condition` is an Expression.

Reply:
The idea is to finally remove this in the subsequent assignment refactor, but at this point, it is still entangled with other functions. I left it just as an intermediate step to finally remove it.

Reply:
Ok.  Then it might be good to add this information as a comment, just in case it is inadvertently forgotten.